src/data/list/basic.lean

```diff
@@ -1287,6 +1287,17 @@ theorem nth_update_nth_ne (a : α) {m n} (l : list α) (h : m ≠ n) :
   nth (update_nth l m a) n = nth l n :=
 by simp only [update_nth_eq_modify_nth, nth_modify_nth_ne _ _ h]
 
+@[simp] lemma update_nth_nil (n : ℕ) (a : α) : [].update_nth n a = [] := rfl
+
+lemma update_nth_comm (a b : α) : Π {n m : ℕ} (h : n ≠ m) (l : list α),
```

Comment:
It's tempting to swap the h and l arguments to match the order they appear in the statement and the lemma above